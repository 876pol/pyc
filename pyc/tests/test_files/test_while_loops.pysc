int main() {
    int i = 0;
    while (i < 5) {
        print((string) i);
        i += 1;
    }
    print("\n");

    i = 0;
    while (i < 10) {
        if (i == 6) break;
        print((string) i);
        i += 2;
    }
    print("\n");

    i = 0;
    while (i < 10) {
        if (i % 2 == 0) {
            i += 1;
            continue;
        }
        print((string) i);
        i += 1;
    }
    print("\n");

    i = 1;
    do {
        i *= -2;
        if (i == -8) continue;
        print((string) i);
    } while (i != 16);
    print("\n");
    return 0;
}