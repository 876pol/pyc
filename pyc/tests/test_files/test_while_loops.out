01234
024
13579
-2416
